js/src/react/pageElements/Header/Header.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import InfoBlock from "../../components/InfoBlock/InfoBlock";
+import ErrorBlock from "../../components/ErrorBlock/ErrorBlock";
```

Comment:
Need to implement frontend test for the new component.

Reply:
This changes will be in next iteration.